### PR TITLE
Multiple additions for transmitter users

### PR DESCRIPTION
**Pull request by adregner**
Just some miscellaneous additions of new support to use the library and sACN the way I wanted to for a project.  Hopefully the individual commits and code is self-explanatory, happy to discuss anything more.

### CODE REVIEW

**Comment by TheIncredibleMulk** [comment]
@adregner I took a glace over the code and it makes sense to me overall, any idea if Hundemeier is still around and paying attention to this repo?
 @Hundemeier Is there something I can do to help move this PR along?

Thanks,
TheMulk

File: sacn/transmitter.go [hunk 1]
```diff
@@ -189,6 +195,14 @@ func (t *Transmitter) sendOut(server *net.UDPConn, universe uint16) {
 	}
 }
 
+func (t *Transmitter) SetKeepAlive(interval time.Duration) {
```
Comment (Hundemeier) [inline sacn/transmitter.go:198]: Please add a comment preceding the function declaration describing what this function does. I suggest using your commit message.

File: sacn/transmitter.go [hunk 2]
```diff
@@ -189,6 +195,14 @@ func (t *Transmitter) sendOut(server *net.UDPConn, universe uint16) {
...
+	t.keepAliveInterval = interval
+}
+
+func (t *Transmitter) SetPriority(prio byte) {
```
Comment (Hundemeier) [inline sacn/transmitter.go:202]: Same as the `SetKeepAlive` function. Please add a comment.

**Comment by Hundemeier** [comment]
Sorry for not replying in time to this PR.
I had broken notfication settings in september and simply didn't notice this PR.

I left two small remarks, but other than that, the changes seem reasonable to me as well.

**Comment by Hundemeier** [comment]
Since there is interest in getting this issue/pull request further along, I decided to merge it even without any comments. But I will not create a new release just yet.

@TheIncredibleMulk created these additional comments (and corrected spelling mistakes) in adregner/go-sacn#1 and I am open to accept @TheIncredibleMulk 's PR on this repo.

**Pull request merged**